x-pack/plugins/cases/public/components/create/form_context.tsx

```diff
@@ -27,6 +27,7 @@ const initialCaseValue: FormProps = {
   connectorId: 'none',
   fields: null,
   syncAlerts: true,
+  selectedOwner: '',
```

Comment:
nit: I think is best to set it to `null` to be more declarative about the absence of the value.

Reply:
done!